### DISCUSSION TITLE
Serious Educational Potential

### DISCUSSION THREAD

**Discussion by evilkillerfiggin** [SebLague/Geographical-Adventures#106] [Ideas]
Hello,

I am recommending this game to all homeschoolers or similarly interested parents I meet. 

I think this is an amazing game for teaching geography to children (and adults - I've learned a hell of a lot already.) It is educational without being didactic, stressful or condescending. Actually, this game is the best of its type and has become the model I compare all other educational games against.

If this project is still being worked on, I believe a few extra features could improve it substantially. I am not in a position to add these myself so I can only suggest them and hope you read this and decide they're a good idea.

Suggestions:

1. Make the country's flags appear next to their names so we learn those as we play. (Flags on balloons looks very pretty but you never see the whole flag at once, so you don't really learn as fast.)
	-> I realise this could ruin the nice calm colour scheme. But for educational purposes, the benefit is well worth the cost. So maybe this is better as a togglable setting?

2. Add the ability to show cities on the map. Currently (just from people I've shown this to) people quickly learn the countries, but they never really get on to learning the cities.
	-> My suggestion is to have a second mode within the map view:
	: press 'M' for map -> see whole view
	: hold 'shift' during map view -> view zooms in, and cities appear
	: release 'shift' to zoom out again, or move around the map view while still zoomed in

3. Add sub-country borders (states, provinces, prefectures, counties, etc.) 
	-> Quite apart from breaking up huge regions like China, India and the USA, (which players don't have a chance of scoring well on unless they've already learned about those places from somewhere else) I believe this extra detail would also make countries come much more alive to players. They'd also end up with "local knowledge" that few outside the country itself would expect them to have.
	-> I hesitated to suggest it though, because while you clearly already have the data for city locations conveniently to hand, I don't know if data on states/provinces/etc is as quickly and conveniently available to you. I also haven't thought the UI through as much.

4. This is a silly throw-away idea, but it occurred to me that if the data on countries,flags,etc could be loaded in externally, you could allow people to contribute their own packages for different time periods. So you could, for example, fly over Europe during WW2 and see how the borders change. Or watch the expansion of Rome.
	-> As I said, silly.

You say in your video that this game was just about testing your knowledge - but I think that with a few improvements this game could become the thing that all children everywhere first learn their geography from.

I think this is a really good project and you should be proud of it.

**Comment by SebLague** [comment]
Hi, thanks for the great feedback and ideas. I'm currently focusing on other projects, but I do plan to come back to this one, so I'll keep your suggestions in mind!

**Reply by GeorgeS2019** [reply]
@SebLague 
I am attempting to bring this high educational value to Godot community
https://github.com/V-Sekai/unidot_importer/issues/22#issue-1799756099

Please advise if this is fine with you and how to make this study within your acceptable guide
